src/ILCompiler.WebAssembly/src/ILCompiler.WebAssembly.csproj

```diff
@@ -10,14 +10,21 @@
     <NoWarn>8002</NoWarn>
     <OutputPath>$(BaseOutputPath)$(OSPlatformConfig)/tools</OutputPath>
   </PropertyGroup>
+  
   <ItemGroup>
     <PackageReference Include="LLVMSharp">
-      <Version>3.9.1-rc3</Version>
+      <Version>5.0.0</Version>
     </PackageReference>
-    <PackageReference Include="libLLVM">
-      <Version>3.9.1</Version>
+    <PackageReference Include="libLLVM4">
```

Comment:
@jkotas, is it ok for ILC to take a dependency on @christianscheuer's build of libLLVM?

Reply:
I will ask around.

Reply:
Yea, I figured that this could be a potential security/maintenance issue.
If you need to build it yourselves, here are the instructions (on OSX 10.12.6):
Run `brew install llvm@4`
Use code from here:
https://github.com/christianscheuer/LLVMSharp/tree/llvm-4/tools/libLLVM
run `./build.sh` go to nuget dir and run `nuget pack`

The windows binaries here are just directly copied from the old 3.9.1 libLLVM official build.